src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -114,28 +115,46 @@
  * }
  * }
  * </blockquote>
- * <p>
- * Specifying a pattern for ChoiceFormat objects is fairly straightforward.
- * For example:
+ * Would output the following:
+ * <blockquote>
+ * <pre>{@code
+ * There are no files on ADisk
+ * There is one file on ADisk
+ * There are 2 files on ADisk
+ * There are 3 files on ADisk
+ * }</pre>
+ * </blockquote>
+ *
+ * <h2><a id="patterns">Patterns</a></h2>
+ * A {@code ChoiceFormat} pattern has the following syntax:
+ * <blockquote><pre>
+ * <i>Pattern:</i>
+ *         SubPattern *("|" SubPattern)
+ * <i>SubPattern:</i>
+ *         Limit Relation Format
+ * <i>Limit:</i>
+ *         {@code String} that can be parsed as a {@code double} / "&infin;" ({@code U+221E}) / "-&infin;" (-{@code U+221E}).
+ * <i>Relation:</i>
+ *         "#" / "&lt;" / "&le;" ({@code U+2264})
+ * <i>Format:</i>
+ *         {@code String}
+ * </pre></blockquote>
+ *
+ * <i>Note:The relation &le; is not equivalent to &lt;&equals;</i>
+ *
+ * <p>Below is an example of constructing a ChoiceFormat with a pattern:
  * <blockquote>
  * {@snippet lang=java :
  * ChoiceFormat fmt = new ChoiceFormat(
  *      "-1#is negative| 0#is zero or fraction | 1#is one |1.0<is 1+ |2#is two |2<is more than 2.");
- * System.out.println("Formatter Pattern : " + fmt.toPattern());
  *
  * System.out.println("Format with -INF : " + fmt.format(Double.NEGATIVE_INFINITY));
- * System.out.println("Format with -1.0 : " + fmt.format(-1.0));
- * System.out.println("Format with 0 : " + fmt.format(0));
- * System.out.println("Format with 0.9 : " + fmt.format(0.9));
- * System.out.println("Format with 1.0 : " + fmt.format(1));
- * System.out.println("Format with 1.5 : " + fmt.format(1.5));
- * System.out.println("Format with 2 : " + fmt.format(2));
- * System.out.println("Format with 2.1 : " + fmt.format(2.1));
+ * // ...
```

Comment:
I felt it was redundant to have both the long lines of println and output, so I commented out the println(s) except for the ones of interest, which are `Double.NEGATIVE_INFINITY`, `Double.POSITIVE_INFINITY`, and `Double.NaN`.

However, you're right that commenting it out would not print the below result. I have updated it so that the println and output lines are combined, to provide a more concise example.